assets/src/edit-story/components/panels/design/videoAccessibility/videoAccessibility.js

```diff
@@ -60,14 +62,15 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
     DEFAULT_RESOURCE
   );
   const alt = getCommonValue(selectedElements, 'alt', resource.alt);
+  const { height, width } = resource;
```

Comment:
Why not using `getCommonValue` anymore? What happens if multiple elements are selected?

Reply:
What happens here is gets the height and widget of the element and not the resource. As alt / poster do not exist on the element, it finds them in resource. Height and width are different.